### PR TITLE
filetop: fix params keys

### LINKED ISSUE #562: [FIX] filetop: Make params keys coherent
https://github.com/kinvolk/inspektor-gadget/pull/551#discussion_r824516983

### PR DESCRIPTION
**Pull request by eiffel-fl**
Hi.

This PR fixes #562.

Best regards.

### CODE REVIEW

File: cmd/kubectl-gadget/filetop.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ var filetopCmd = &cobra.Command{
 
 func init() {
 	filetopCmd.Flags().IntVarP(&maxRows, "maxrows", "r", 20, "Maximum rows to print")
-	filetopCmd.Flags().StringVarP(&sortByStr, "sort", "", "rbytes", "Sort column")
+	filetopCmd.Flags().StringVarP(&sortByStr, "sort", "", "rbytes", fmt.Sprintf("Sort column, possibles values are: %s", strings.Join(types.SortBySlice, ", ")))
```
Comment (mauriciovasquezbernal) [inline cmd/kubectl-gadget/filetop.go:104]: IMO "possibles values are" is too verbose, the following is a bit shorter and understandable as well. 

```suggestion
	filetopCmd.Flags().StringVarP(&sortByStr, "sort", "", "rbytes", fmt.Sprintf("Sort column: (%s)", strings.Join(types.SortBySlice, ", ")))
```

File: pkg/gadgets/filetop/gadget.go [hunk 2]
```diff
@@ -118,34 +121,34 @@ func (t *Trace) Start(trace *gadgetv1alpha1.Trace) {
...
 				return
 			}
 		}
 
-		if val, ok := params["sortby"]; ok {
+		if val, ok := params[types.SortByParam]; ok {
 			sortBy, err = types.ParseSortBy(val)
 			if err != nil {
-				trace.Status.OperationError = fmt.Sprintf("%q is not valid for sortby", val)
+				trace.Status.OperationError = fmt.Sprintf("%q is not valid for %q: %v", val, types.SortByParam, err)
```
Comment (mauriciovasquezbernal) [inline pkg/gadgets/filetop/gadget.go:143]:
```suggestion
				trace.Status.OperationError = fmt.Sprintf("%q is not valid for %q: %s", val, types.SortByParam, err)
```

?

File: pkg/gadgets/filetop/gadget.go [hunk 3]
```diff
@@ -118,34 +121,34 @@ func (t *Trace) Start(trace *gadgetv1alpha1.Trace) {
...
 				return
 			}
 		}
 
-		if val, ok := params["all_files"]; ok {
+		if val, ok := params[types.AllFilesParam]; ok {
 			allFiles, err = strconv.ParseBool(val)
 			if err != nil {
-				trace.Status.OperationError = fmt.Sprintf("%q is not valid for all_files", val)
+				trace.Status.OperationError = fmt.Sprintf("%q is not valid for %q: %v", val, types.AllFilesParam, err)
```
Comment (mauriciovasquezbernal) [inline pkg/gadgets/filetop/gadget.go:151]: ditto.

**Review by mauriciovasquezbernal — COMMENTED** [review]
Just some comments, LGTM otherwise. Thanks for fixing it!

File: cmd/kubectl-gadget/filetop.go (hunk 1 above)
Comment (mauriciovasquezbernal) [reply cmd/kubectl-gadget/filetop.go:104]: This is how it's printed in `docs/gadgets/filetop.md` too, I think those two output should be aligned as much as possible.

Comment (eiffel-fl) [reply cmd/kubectl-gadget/filetop.go:104]: Thinking to it, it is also the way output possibles values are printed:

```
-o, --output string          Output format (columns, json, custom-columns). (default "columns")
```

So let's stick to this one, it we want to change it, we would do it later.

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**